tests/kernel/embedding.cc

```diff
@@ -0,0 +1,27 @@
+/*! @copyright (c) 2022-present Skolkovo Institute of Science and Technology
+ *                              (Skoltech), Russia. All rights reserved.
+ *                 2023-present Artificial Intelligence Research Institute
+ *                              (AIRI), Russia. All rights reserved.
+ *
+ * NNTile is software framework for fast training of big neural networks on
+ * distributed-memory heterogeneous systems based on StarPU runtime system.
+ *
+ * @file tests/kernel/embedding.cc
+ * Placeholder for embedding kernel test
+ *
+ * @version 1.1.0
+ * */
+
+#include "nntile/kernel/embedding.hh"
+#include "../testing.hh"
+#include <iostream>
+
+using namespace nntile;
+using namespace nntile::kernel;
+
+int main(int argc, char **argv)
+{
+    // This is a placeholder test.
+    std::cout << "STUB: This is a placeholder for the embedding kernel test." << std::endl;
+    return 0;
+}
```

Comment:
add /n after the line

Reply:
You are playing with the wrong newline. Your task was to add newline character to the end of the file, not to the std::cout statement. Revert std::cout statement and properly add newline character to the end of the file.